CinC2016Challenge_PCG/examples/nupic2016/model/description.py

```diff
@@ -0,0 +1,493 @@
+
+            'maxAge': 0,
+
+            # Minimum number of active synapses for a segment to be considered
+            # during search for the best-matching segments.
+            # None=use default
+            # Replaces: tpMinThreshold
+            'minThreshold': 12,
+
+            # Segment activation threshold.
+            # A segment is active if it has >= tpSegmentActivationThreshold
+            # connected synapses that are active due to infActiveState
+            # None=use default
+            # Replaces: tpActivationThreshold
+            'activationThreshold': 12,
+
+            'outputType': 'normal',
+
+            # "Pay Attention Mode" length. This tells the TP how many new
+            # elements to append to the end of a learned sequence at a time.
+            # Smaller values are better for datasets with short sequences,
+            # higher values are better for datasets with long sequences.
+            'pamLength': PAM,
+        },
+
+        'clParams': {
+            # Classifier implementation selection.
+            'implementation': 'cpp',
+            
+            'regionName' : 'CLAClassifierRegion',
+
+            # Classifier diagnostic output verbosity control;
+            # 0: silent; [1..6]: increasing levels of verbosity
+            'clVerbosity' : 0,
+
+            # This controls how fast the classifier learns/forgets. Higher values
+            # make it adapt faster and forget older patterns faster.
+            'alpha': 0.0001,
+
+            # This is set after the call to updateConfigFromSubConfig and is
+            # computed from the aggregationInfo and predictAheadTime.
+            'steps': '1',
+
+
+        },
+
+        'anomalyParams': {
+           'mode': AN_MODE, # pure(=default) / weighted / likelihood
+           'slidingWindowSize': AN_WINDOW, # >=0 / None
+        },
+
+        'trainSPNetOnlyIfRequested': False,
+    },
+
+
+  'predictionSteps': [1],
+  'predictedField': 'consumption',
+  'numRecords': LIMIT,
+}
+# end of config dictionary
+
+
+# Adjust base config dictionary for any modifications if imported from a
+# sub-experiment
+updateConfigFromSubConfig(config)
+
+
+# Compute predictionSteps based on the predictAheadTime and the aggregation
+# period, which may be permuted over.
+if config['predictAheadTime'] is not None:
+  predictionSteps = int(round(aggregationDivide(
+      config['predictAheadTime'], config['aggregationInfo'])))
+  assert (predictionSteps >= 1)
+  config['modelParams']['clParams']['steps'] = str(predictionSteps)
+
+
+# Adjust config by applying ValueGetterBase-derived
+# futures. NOTE: this MUST be called after updateConfigFromSubConfig() in order
+# to support value-getter-based substitutions from the sub-experiment (if any)
+applyValueGettersToContainer(config)
+control = {
+  # The environment that the current model is being run in
+  "environment": 'opfExperiment', #'nupic',
+
+ "tasks":[
+ {
+  "taskLabel": "train",
```

Comment:
I'm sorry @breznak but I never use `run_opf_experiment.py`. I found that interface too confusing. Maybe @scottpurdy can help you.